__tests__/components/AnonymousChat/child/ChatItemTargetText.test.tsx

```diff
@@ -34,7 +41,8 @@ const renderChatItemTargetText = (isContinous = false) =>
     chatType: SIGNED,
     data: expectedReplyPreview.payload,
     isContinuous: isContinous,
-    time: timestamp
+    time: timestamp,
+    attachments: []
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces an `attachments` field in the chat item object. Ensure that all functions and components that use this object are updated to handle this new field appropriately. If the `attachments` field is optional, make sure to check its existence before accessing it.
```diff
+    attachments: []
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->